test/reg.jl

```diff
@@ -0,0 +1,51 @@
+@testset "Binomial GLM in bsrs" begin
+    rename!(bsrs.data, Symbol("sch.wide") => :sch_wide)
+    bsrs.data.sch_wide = ifelse.(bsrs.data.sch_wide .== "Yes", 1, 0)
+    model = svyglm(@formula(sch_wide ~ meals + ell), bsrs, Binomial())
```

Comment:
Most importantly, you need some tests using jackknife replicates. We only have bootstrap replicates. You will face problems with jackknife.